packages/components/src/utils/property-checkers/check-url.ts

```diff
@@ -1,15 +1,14 @@
-export function checkUrl<T extends { host: HTMLElement }>(
-  component: T,
-  prop: keyof T,
-  customMessage?: string,
-) {
+import { isValueEmpty } from '../is-value-empty';
+
+export function checkUrl<T extends { host: HTMLElement }>(component: T, prop: keyof T) {
   const componentName = component.host.localName;
   const value = component[prop];
 
-  const defaultMessage = `The prop \`${String(
-    prop,
-  )}\` of the \`${componentName}\` component is invalid.`;
-  const message = customMessage || defaultMessage;
+  const message = `The prop \`${String(prop)}\` of the \`${componentName}\` component is invalid.`;
+
+  if (isValueEmpty(value)) {
+    return;
+  }
```

Comment:
Thanks to your last changes, this check can be removed from the `check*` functions:
```suggestion
```